software/gui/gui.sh

```diff
@@ -169,75 +120,102 @@ install_linux() {
           qml-module-qtquick-controls2 \
           qml-module-qtmultimedia \
           pulseaudio \
+          python-pip \
           xvfb \
           bear \
           cppcheck \
+          gcovr \
           lcov \
           clang-tidy
 }
 
-checks_pre() {
-  #Should happen in the build directory
-  cppcheck -ithird_party -ibuild .
+configure_conan() {
+  conan profile new --detect default
+  conan profile update settings.compiler.libcxx=libstdc++11 default
+  conan remote add conan-center https://conan.bintray.com False -f
+  conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan False -f
 }
 
-checks_post() {
-  #Should happen in the build directory
-
-  cppcheck --project=compile_commands.json \
-           -i ../../src/third_party \
-           -i ../../../common/third_party \
+run_cppcheck() {
+  create_clean_directory  build/cppcheck
+  cppcheck --enable=all --std=c++17 --inconclusive --force --inline-suppr --quiet \
+           -I ../common/generated_libs/network_protocol \
+           -I ../common/third_party/nanopb \
+           -I ../common/libs/units \
+           -ibuild -icmake-build-stm32 -isrc/third_party \
            .
 
+#           --project=build/compile_commands.json \
+#           --xml --output-file=build/cppcheck/report.xml \
+#
+#  cppcheck-htmlreport --file=build/cppcheck/report.xml \
+#                      --title="Ventilator GUI" \
+#                      --report-dir=build/cppcheck --source-dir=.
```

Comment:
Do we want to keep this commented out block? (We might...)

Reply:
Yeah, I think I was trying to get some html reports generated. Might still want to try this later?